docs/oauth2-oidc/custom-login-consent/flow.mdx

```diff
@@ -3,61 +3,55 @@ id: flow
 title: User login and consent flow
 ---
 
-OAuth2 and OpenID Connect are powerful protocols for securing your applications and APIs. Ory OAuth2 & OpenID Connect is a highly
-flexible and customizable implementation of these protocols that allows you to easily bring your own user database, login UI,
-consent logic, and consent UI.
-
-This guide will walk you through the process of customizing Ory OAuth2 & OpenID Connect to work with your user database, login UI,
-and consent logic and UI. We'll cover everything you need to know, including how to redirect users to your login endpoint, how to
-fetch and process login and consent requests, and how to use the SDK to accept or reject these requests.
+This guide will walk you through the process of customizing Ory OAuth2 and OpenID Connect to work with your user database, login
```

Comment:
I thought we were going to use & in the product name? So do we not leave this as "Ory OAuth2 & OpenID Connect"? I agree that it's kinda ugly, but that's how I left it in the sections that I worked on!

Reply:
As discussed, we're gonna use the ampersand-less version of the name from now on :)